pym/bob/generators/QtCreatorGenerator.py

```diff
@@ -200,11 +206,11 @@ def qtProjectGenerator(package, argv, extra):
     getCheckOutDirs(package, excludes, dirs)
     if not projectName:
         # use package name for project name
-        projectName = package.getName()
+        projectName = package.getName().replace('::', '__')
     if not destination:
         # use package name for project directory
-        destination = os.path.join(os.getcwd(), "projects", projectName)
-        destination = destination.replace(':', '_')
+        destination = os.path.join(os.getcwd() if not isWindows() else os.popen('pwd -W').read().strip(), "projects", projectName)
```

Comment:
Can you replace all `os.getcwd() if not isWindows() else os.popen('pwd -W').read().strip()` instances by invoking a helper?, e.g.:

```python
def cwd():
    return os.getcwd() if not isWindows() else os.popen('pwd -W').read().strip()

...

        destination = os.path.join(cwd(), "projects", projectName)
```

That'll remove the code duplication and make it more readable.